src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 0 50px;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  height: 60px;
+  background-color: hsl(0, 0%, 100%);
+  /* padding: 0 50px; */
+}
+
+.link__logo {
+  display: flex;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  /* padding: 50px; */
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: space-between;
+  /* height: 60px; */
+  align-items: center;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 23px 0;
+  color: hsl(0, 0%, 0%);
+}
+
+.is-active {
+  color: hsl(193, 100%, 43%);
+  position: relative;
+
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  width: 37px;
```

Comment:
Do not hardcode width. Try setting `is-active` style to other links and you will see why.

instead set the width to `100%` so it will fit any element.